conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureExceptions.java

```diff
@@ -186,17 +192,14 @@ private static void error(HttpServerExchange exchange, Error error) {
         writeResponse(exchange, Optional.empty(), ErrorType.INTERNAL.httpErrorCode());
     }
 
-    private static void writeResponse(
-            HttpServerExchange exchange, Optional<SerializableError> maybeBody, int statusCode) {
+    private static void writeResponse(HttpServerExchange exchange, Optional<ConjureError> maybeBody, int statusCode) {
         // Do not attempt to write the failure if data has already been written
         if (!isResponseStarted(exchange)) {
             exchange.setStatusCode(statusCode);
-            if (maybeBody.isPresent()) {
```

Comment:
was this intentionally deleted? I think the empty case will produce an exception and log it at info as `log.info("Failed to write error response", e);`

Reply:
Sorry, this was a typo.